sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -8,10 +8,14 @@
 #include "sw/device/lib/usb_controlep.h"
 #include "sw/device/lib/usb_simpleserial.h"
 #include "sw/device/lib/usbdev.h"
+#include "usbdev_regs.h"  // Generated.
 
 // These just for the '/' printout
 #define USBDEV_BASE_ADDR 0x40020000
-#include "usbdev_regs.h"  // Generated.
+
+#define USBDEV_REG_DEF(id, rname) (REG_DEF(USBDEV, id, rname))
```

Comment:
I'd get rid of these `_REG_DEF` macros. They don't really save much on typing and are extra, unnecessary indirection.

Reply:
I agree, and will update the change if we decide to go with this PR.